### PR TITLE
Remove dry-run

### PR DESCRIPTION
**Pull request by MUzairS15**
Signed-off-by: MUzairS15 <muzair.shaikh810@gmail.com>

**Description**

This PR remove dry-run from patterns when they are fetched causing slow behaviour.

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by Revolyssup — APPROVED** [review]

File: server/handlers/meshery_pattern_handler.go [hunk 1]
```diff
@@ -278,14 +278,14 @@ func (h *Handler) GetMesheryPatternsHandler(
...
 		return
 	}
-	mc := NewContentModifier(token, provider, prefObj, user.UserID)
-	//acts like a middleware, modifying the bytes lazily just before sending them back
-	err = mc.AddMetadataForPatterns(r.Context(), &resp)
+	// mc := NewContentModifier(token, provider, prefObj, user.UserID)
+	// //acts like a middleware, modifying the bytes lazily just before sending them back
+	// err = mc.AddMetadataForPatterns(r.Context(), &resp)
 	if err != nil {
 		fmt.Println("Could not add metadata about pattern's current support ", err.Error())
```
Comment (leecalcote) [inline server/handlers/meshery_pattern_handler.go:290]: Yikes `fmt.Println`

Comment (leecalcote) [reply server/handlers/meshery_pattern_handler.go:290]: @Revolyssup will you please add comments that describe the behavior that these lines have (i.e. dry run)?

File: server/internal/graphql/resolver/pattern.go [hunk 2]
```diff
@@ -21,8 +21,8 @@ func (r *Resolver) fetchPatterns(ctx context.Context, provider models.Provider,
 		return nil, err
 	}
 
-	mc := handlers.NewContentModifier(tokenString, provider, prefObj, user.UserID)
-	err = mc.AddMetadataForPatterns(ctx, &resp)
+	// mc := handlers.NewContentModifier(tokenString, provider, prefObj, user.UserID)
```
Comment (leecalcote) [inline server/internal/graphql/resolver/pattern.go:24]: It's not obvious that this code invokes a dry-run. Will you please add comments, describing this?

Comment (MUzairS15) [reply server/internal/graphql/resolver/pattern.go:24]: This func `AddMetadataForPatterns` process each pattern file fetched, https://github.com/meshery/meshery/blob/a190c7ecb88ac8fd32e2d2fd6f66dddb6b9c57a7/server/handlers/content_modifier.go#L53

and uses pattern engine to validate it (dry run) 
https://github.com/meshery/meshery/blob/a190c7ecb88ac8fd32e2d2fd6f66dddb6b9c57a7/server/handlers/content_modifier.go#L81

and finally add the results in the patterns fetched from the provider.

Comment (leecalcote) [reply server/internal/graphql/resolver/pattern.go:24]: Thank you, @MUzairS15. Will you please add these comments in the code, too?

**Pull request merged**